week10/src/Firebase.js

```diff
@@ -0,0 +1,24 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: "AIzaSyD3C5k84wDbpBqcbGkdSYAAPopIu8sdzs8",
+  authDomain: "focus-app-cac16.firebaseapp.com",
+  projectId: "focus-app-cac16",
+  storageBucket: "focus-app-cac16.appspot.com",
+  messagingSenderId: "707105423759",
+  appId: "1:707105423759:web:edd26053462aaa72c54bfa",
+  measurementId: "G-C9EX4KGQG5"
+};
```

Comment:
do not hard code api key, https://medium.com/how-to-react/using-env-file-in-react-js-b2714235e77e